app/src/main/java/com/spm/vasylyshyn/counterdisplayapp/activity/LoginActivity.kt

```diff
@@ -34,9 +36,8 @@ class LoginActivity : AppCompatActivity() {
                 ) { dialog, whichw -> dialog.dismiss() }
                 alertDialog.show()
             } else {
-                val ip = IP.ip
                 val retrofit = Retrofit.Builder()
-                    .baseUrl("http://${ip}:8080/api/")
+                    .baseUrl(URL_PATH)
                     .addConverterFactory(GsonConverterFactory.create())
                     .build()
 //                val loginRequest:LoginRequest = LoginRequest("vasiliwin@gmail.com", "vova")
```

Comment:
Remove commented out code

Reply:
Remove all commented out code and rename URL_PATH